docs/building/menus.md

```diff
@@ -0,0 +1,66 @@
+A menu represents a type of response ONEm sends back to the user and it allows the user to select certain options you put in the menu.
+
+It is composed by a **header**, a **body** and a **footer**. The body can contain **option** or **content** items.
+
+When an option is being accessed, ONEm platform will perform an HTTP request to the callback path you set.
+
+## Json Structure
+
+Your server must return a JSON response with a structure similar to the one below:
+
+```
+{'body': [{'description': 'New todo',
+           'method': 'GET',
+           'nextRoute': '/task/create/',
+           'type': 'option'},
+          {'description': 'Done(0)',
+           'method': 'GET',
+           'path': '/task/list/done/',
```

Comment:
What's the difference between `path` and `nextRoute` ?

Reply:
No difference, same thing. Personal preference, snake case vs camel case typing.

Also `path` is a more appropriate naming from my pov, rather than `route`